pkg/web/mobileappHandler.go

```diff
@@ -16,6 +16,18 @@ type MobileAppHandler struct {
 	tokenClientBuilder mobile.TokenScopedClientBuilder
 }
 
+// OauthHandler handle oauth actions
```

Comment:
I think we should move the oauth handler to its own handler under the web directory.